dashboard/app/views/admin_standards/import_standards.html.haml

```diff
@@ -0,0 +1,57 @@
+:ruby
+  courses = [
+    ["", ""],
+    ["Course A 2019", "coursea-2019"],
+    ["Course B 2019", "courseb-2019"],
+    ["Course C 2019", "coursec-2019"],
+    ["Course D 2019", "coursed-2019"],
+    ["Course E 2019", "coursee-2019"],
+    ["Course F 2019", "coursef-2019"],
+    ["Pre-Express 2019", "pre-express-2019"],
+    ["Express 2019", "express-2019"]
+  ]
+
+%h2
+  Import Standards from Curriculum Builder
+
+%p
+  The associations between standards and lessons are stored in Curriculum Builder and used to show standards alignment by course. For example:
+%a{href: 'curriculum.code.org/csf-19/standards/'}
+  CSF Standards Alignment
+
+%br
+%br
+
+%p
+  Information about CSTA standards alignment is also used on the standards view of the progress tab of the Teacher Dashboard on Code Studio for CSF courses. Select a course to import standards associated with that course's lessons.
+
+%h3 Select a Course
+= form_tag url_for(action: 'import_standards'), id: 'standards-form', method: 'post', class: 'form-inline', enforce_utf8: false do
+  = select_tag(:course, options_for_select(courses), id: "select")
+  %br
+  %br
+  %button.primary.padded-button#import-standards="Import Standards"
+
+%p#data-spew
+
+:javascript
+  $(document).ready(function() {
+    $('#import-standards').click(function (e) {
```

Comment:
Please move is to an endpoint in our apps/ directory:
1. extract this JS into a file named apps/src/sites/studio/pages/shared/import_standards.js
2. add and entry for it here: https://github.com/code-dot-org/code-dot-org/blob/ca45f74d8c29d1a87cbb87910496300fbdca839f/apps/Gruntfile.js#L583-L587
`shared/import_standards: './src/sites/studio/pages/shared/import_standards.js'`
3. include it into this page with the following code in place of `:javascript`:
```
- content_for(:head) do
  %script{ src: webpack_asset_path('js/shared/import_standards.js')
```
This will allow the js to get linted so that we are less likely to break this code.

I know we think this code will go away, but these things tend to live longer than we expect and need to be updated more often than we expect... so it's best to take some basic precautions.

Reply:
Working on extracting now. Why shared in `apps/src/sites/studio/pages/shared/import_standards.js`? I thought that was for JS shared between Pegasus and Dashboard, like the header?

Reply:
sorry, I don't know how I ended up with "shared". the path should match the filename to the haml file, in this case admin_standards/import_standards no shared/import_standards .

Reply:
Got it. Just wanted to double check I wasn't missing something. Updated ✅ Thanks!